src/plugins/vis_type_timelion/public/helpers/timelion_request_handler.ts

```diff
@@ -59,7 +58,7 @@ export interface TimelionSuccessResponse {
   sheet: Sheet[];
   stats: Stats;
   visType: string;
-  type: KIBANA_CONTEXT_NAME;
```

Comment:
This was the only place in Kibana this type was used, and you can derive it from `KibanaContext` already -- so rather than keeping it in the `data` plugin contract, I removed it and updated accordingly.